### ISSUE TITLE
UnboundLocalError: local variable 'publication' referenced before assignment

### ISSUE THREAD

**Issue by tpoisot** [sjpfenninger/citebibtex#9]
When the package tries to load, I get the following error.

Using Linux Mint, ST 3, and AcademicMarkdown

~~~ python
Package Control: Skipping automatic upgrade, last run at 2015-06-08 22:08:18, next run at 2015-06-08 23:08:18 or after
Traceback (most recent call last):
  File "/home/tpoisot/.config/sublime-text-3/Packages/CiteBibtex/CiteBibtex.py", line 27, in <lambda>
    sublime.set_timeout_async(lambda: self.update_refs(global_file), 0)
  File "/home/tpoisot/.config/sublime-text-3/Packages/CiteBibtex/CiteBibtex.py", line 155, in update_refs
    self.refs[ref_file] = [self.get_item(refs[i]) for i in refs]
  File "/home/tpoisot/.config/sublime-text-3/Packages/CiteBibtex/CiteBibtex.py", line 155, in <listcomp>
    self.refs[ref_file] = [self.get_item(refs[i]) for i in refs]
  File "/home/tpoisot/.config/sublime-text-3/Packages/CiteBibtex/CiteBibtex.py", line 117, in get_item
    if publication == title:
UnboundLocalError: local variable 'publication' referenced before assignment
~~~

**Comment by sjpfenninger** [comment]
Strange, I'll investigate

**Comment by sjpfenninger** [comment]
This should be fixed in v1.2.2 soon available via Package Control, let me know if there are still problems after updating!
